AAInstrumentation/unique_hashes.py

```diff
@@ -0,0 +1,67 @@
+from core import AAInstrumentationDriver, register_arguments
+from dataclasses import dataclass, field
+import sys
+from pathlib import Path
+from typing import Dict, List
+import random
+
+
+@dataclass
+class UniqueHashesDriver(AAInstrumentationDriver):
+    def get_num_unique_hashes(
```

Comment:
What is this thing doing? 

Is it sampling random sequences and computing all the unique ones? Could you add a comment describing this?

Also would it make sense to at the random seed as an argument?


And would it also make sense to return the candidates themselves? I am wondering, e.g., how different are they? Or whether given one candidate, can you compute some kind of "equivalence class" of AA sequences?

Reply:
I added a docstring and a seed field. Notice that this has the issue with sequences that look different but are the same atm (becuase there are less queries than the length of the sequence). I decided to leave this for a future PR.

I think, if we wanted to generate more stats, we can easily extend this. I agree that it might be interesting to track which sequences lead to the same result and then try to figure out why/what happened. Though, I would also leave this for a future PR.